epictrack-web/src/components/workPlan/status/StatusForm.tsx

```diff
@@ -19,7 +19,23 @@ const StatusForm = () => {
   const [description, setDescription] = React.useState<string>("");
   const startDateRef = useRef();
   const { status, onSave, isCloning } = useContext(StatusContext);
-  const { getWorkStatuses } = useContext(WorkplanContext);
+  const { getWorkStatuses, statuses } = useContext(WorkplanContext);
+
+  const getPostedDateMin = () => {
+    if (isCloning) {
+      return dayjs(statuses[0].posted_date);
+    }
+    if (statuses.length === 1 && statuses[0]?.is_approved) {
+      return dayjs("1900-01-01");
+    }
+
+    return dayjs(statuses[1]?.posted_date || "1900-01-01");
```

Comment:
Is this date something that we can reuse elsewhere. If so, lets make it a constant.